src/components/Store/StoreEdit.vue

```diff
@@ -15,7 +15,7 @@
             <q-input
               v-model="edit.name"
               :autofocus="true"
-              autocomplete="off"
+              :autocomplete="off"
```

Comment:
Why does it need a binding instead of a plain static attribute?

Reply:
I will rectify this.